net-misc/ntpsec/ntpsec-9999.ebuild

```diff
@@ -119,25 +117,16 @@ src_configure() {
 		$(use smear	&& echo "--enable-leap-smear")
 		$(use debug	&& echo "--enable-debug")
 	)
-
-	distutils-r1_src_configure
+	python_setup
+	waf-utils_src_configure
 }
 
-python_configure() {
-	waf-utils_src_configure "${myconf[@]}"
-}
-
-python_compile() {
-	waf-utils_src_compile --notests
-}
-
-python_test() {
-	waf-utils_src_compile check
+src_compile() {
+	waf-utils_src_compile
+	(cd build/main&&${PYTHON} -m build -n)
```

Comment:
Using `build` seems off to me. We're going _from_ using distutils-r1 (which can use `gpep517`, which is what we use as our preferred frontend) to using a custom call for `build`.

I think you should try doing `DISTUTILS_USE_PEP517=setuptools`?

Reply:
Probably not today.